### PR TITLE
Enable dpic tests on Windows

### PR DESCRIPTION
**Pull request by snecklifter**

### CODE REVIEW

File: .github/workflows/ci-windows.yml [hunk 1]
```diff
@@ -58,6 +58,12 @@ jobs:
         run: npm install -g bpmn-js-cmd
       - name: Install Bytefield-SVG
         run: npm install -g bytefield-svg
+      - name: Install Dpic
+        run: |
+          mkdir dpic
+          Invoke-WebRequest -Uri "https://ece.uwaterloo.ca/~aplevich/dpic/Windows/dpic.exe" -OutFile dpic.exe
+          mv dpic.exe dpic
```
Comment (slonopotamus) [inline .github/workflows/ci-windows.yml:65]: Possibly you can avoid `mv` by passing `-OutFile dpic\dpic.exe` to `Invoke-WebRequest` on the previous line?

Comment (snecklifter) [reply .github/workflows/ci-windows.yml:65]: @slonopotamus sure

**Comment by snecklifter** [comment]
I've added a powershell retry function to try and work around some of the issues with downloads.
There are almost certainly better ways to do this. :/

**Comment by pepijnve** [comment]
Caching would probably help here as well

**Pull request merged**